CLI/run_configured_solver.py

```diff
@@ -7,21 +7,20 @@
 
 from runrunner.base import Runner
 
-import global_variables as sgh
+import global_variables as gv
```

Comment:
As this is a relative import change to `.global_variables`. Additionally, this type of import does not pave the way to use the CLI  outside of the sparkle base directory. Please change this if possible.

Reply:
I do agree, but I think global_variables should be in the CLI dir at some point (Once we have decoupled the sparkle/ dir from global_variables). the .global_variables currently does not work (Linter/Python does not understand the pathing)

Reply:
Writing ticket